tutorials/xarray-spatial_classification-methods.ipynb

```diff
@@ -0,0 +1,372 @@
+    "    filename = nasadem_file_prefix + lat_string + lon_string + nasadem_content_extension\n",
+    "\n",
+    "    if filename not in nasadem_file_list:\n",
+    "        print(\"Lat/lon {},{} not available\".format(lat, lon))\n",
+    "        filename = None\n",
+    "\n",
+    "    return filename"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c581d16d-c1a8-4cce-ad77-94b9f98ca9da",
+   "metadata": {},
+   "source": [
+    "Finally, use the function defined above to generate a URL pointing to the geodata for the Grand Canyon area:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "074aa0f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grand_canyon_coord = [36.101690, -112.107676]\n",
+    "\n",
+    "url = nasadem_blob_root + get_nasadem_filename(grand_canyon_coord)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9149231c",
+   "metadata": {},
+   "source": [
+    "After retrieving the raw data for the Grand Canyon, use xarray's [open_rasterio](http://xarray.pydata.org/en/stable/generated/xarray.open_rasterio.html) function to load the data into an array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5476b1fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "\n",
+    "img_arr = xr.open_rasterio(url).squeeze().drop(\"band\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b6ae27af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "img_arr.plot.imshow(figsize=(15, 10));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42c128aa",
+   "metadata": {},
+   "source": [
+    "### 2. Classify elevation data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d1f4e54",
+   "metadata": {},
+   "source": [
+    "#### Classify with `natural_breaks()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41759319",
+   "metadata": {},
+   "source": [
+    "Use the [natural breaks](https://xarray-spatial.org/reference/_autosummary/xrspatial.classify.natural_breaks.html) function to classify data with the [Jenks natural breaks classification](http://wiki.gis.com/wiki/index.php/Jenks_Natural_Breaks_Classification) method. This method is designed to distribute data into classes according to clusters that form a \"natural\" group within the data. The algorithm minimizes the average deviation from the class mean while also maximizing the deviation from the means of the other groups. Therefore, it is generally not recommended for data with low variance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1c5d7b05",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datashader.transfer_functions import shade\n",
+    "import matplotlib.pyplot as plt\n",
+    "from xrspatial.classify import natural_breaks\n",
+    "\n",
+    "natural_breaks_agg = natural_breaks(img_arr, num_sample=20000, k=15)\n",
```

Comment:
A bit of guidance for choosing these parameters would be helpful I think.

Reply:
Also, thoughts on mentioning what `natural_breaks_agg` is, before plotting it? Just something brief like 

   Like most methods in xarray-spatial `natural_breaks` returned an `xarray.DataArray`. It contains integers representing ...